api/mesh/v1alpha1/fault_injection.proto

```diff
@@ -8,32 +8,52 @@ import "google/protobuf/duration.proto";
 import "mesh/v1alpha1/selector.proto";
 import "google/protobuf/wrappers.proto";
 
+// FaultInjection defines configuration of faults between dataplanes.
 message FaultInjection {
 
+  // List of selectors to match dataplanes that are sources of traffic.
   repeated Selector sources = 1;
 
+  // List of selectors to match services that are destinations of traffic.
   repeated Selector destinations = 2;
 
+  // Configuration defines several types of faults, at least one delay should be
+  // specified
   message Conf {
 
+    // Defines configuration of the delay
     message Delay {
+      // Percentage of requests on which delay will be injected
       google.protobuf.DoubleValue percentage = 1;
+      // Duration, the value of delay
       google.protobuf.Duration value = 2;
     }
+    // Delay if specified makes source side to experience delay during requests
+    // to the destination
     Delay delay = 1;
 
+    // Defines configuration of the abort
```

Comment:
How about `Abort defines a configuration of not delivering requests to destination service and replacing the responses from destination dataplane by predefined status code`?

By the way - Is a request with to destination service delivered to the service and response is replaced or is it not delivered and Envoy on the destination side just responds with status code?

Reply:
From envoy documentation: 
_If specified, the filter will **abort requests** based on the values in the object. At least abort or delay must be specified._
So I guess aborting requests means "not delivering them to service".

Reply:
Yeah, it makes more sense for requests to be aborted on Envoy's side. Just wanted to make sure.